.github/schemas/source.schema.json

```diff
@@ -0,0 +1,107 @@
+{
+  "$schema": "http://json-schema.org/draft-07/schema#",
+  "type": "object",
+  "properties": {
+    "source-id": {
+      "type": "string"
+    },
+    "config-version": {
+      "type": "integer"
+    },
+    "networks": {
+      "type": "object",
+      "properties": {
+        "kyve-1": {
+          "$ref": "#/definitions/network"
+        },
+        "kaon-1": {
+          "$ref": "#/definitions/network"
+        }
+      },
+      "additionalProperties": false
+    }
+  },
+  "required": [
+    "source-id",
+    "config-version",
+    "networks"
+  ],
+  "additionalProperties": false,
+  "definitions": {
+    "properties": {
+      "type": "object",
+      "properties": {
+        "hex": {
+          "type": "string",
+          "pattern": "^#([a-fA-F0-9]{6}|[a-fA-F0-9]{3})$"
+        },
+        "title": {
+          "type": "string"
+        },
+        "description": {
+          "type": "string"
+        },
+        "logo": {
+          "type": "string"
+        }
+      }
+    },
+    "network": {
+      "type": "object",
+      "properties": {
+        "pools": {
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/pool"
+          }
+        },
+        "integrations": {
+          "type": "object",
+          "additionalProperties": {
+            "$ref": "#/definitions/integration"
+          }
+        },
+        "properties": {
+          "$ref": "#/definitions/properties"
+        }
+      },
+      "additionalProperties": false
+    },
+    "pool": {
+      "type": "object",
+      "properties": {
+        "id": {
+          "type": "integer"
+        },
+        "runtime": {
+          "type": "string"
+        }
+      },
+      "required": [
+        "id"
+      ]
+    },
+    "integration": {
+      "type": "object",
+      "ksync": {
+        "type": "object",
+        "binary_name": {
```

Comment:
`"binary_name"`, `"block_sync_pool"`, `"docs_url"` and `"state_sync_pool"` should be included in `"properties": {}`, where all attributes are required